tests/unit/exceptions/test-class-wc-payments-amount-too-low-exception.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Class WC_Payments_Amount_Too_Low_Exception_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use WCPay\Exceptions\Amount_Too_Low_Exception;
+
+/**
+ * Test cases for Amount_Too_Low_Exception
+ */
+class WC_Payments_Amount_Too_Low_Exception_Test extends WP_UnitTestCase {
+
+	public function test_getMessage() {
+		$e_without_data = new WCPay\Exceptions\Amount_Too_Low_Exception( 'Test', 'wcpay_amount_too_low', 400, [] );
+		$this->assertEquals(
+			__(
+				'The specified amount is less than the minimum amount allowed. Use a higher amount and try again.',
+				'woocommerce-payments'
+			),
+			$e_without_data->getMessage()
+		);
+
+		$error_data = [
+			'currency'   => 'usd',
+			'min_amount' => 60,
+		];
+
+		$e_with_data = new WCPay\Exceptions\Amount_Too_Low_Exception( 'Test', 'wcpay_amount_too_low', 400, $error_data );
+		$this->assertEquals(
+			sprintf(
+				// translators: %1$s is a formatted amount with currency code.
+				__(
```

Comment:
I kept it there for the cases if we start running tests in multiple languages for any reason.

Reply:
I doubt we'll test translations this way because AFAIK translations are handled via WP outside of the plugin. Anyway, this is a minor thing, so leaving decision up to you.

Reply:
True, removed translations in tests for now.